Sources/UIApplication.swift

```diff
@@ -97,3 +97,33 @@ public func nativeProcessEventsAndRender(env: UnsafeMutablePointer<JNIEnv>, view
     CFRunLoopRunInMode(kCFRunLoopDefaultMode, max(0.001, remainingFrameTime / 2), true)
 }
 #endif
+
+extension UIApplication {
```

Comment:
Would be great if you could put both of these extensions into a new file, something like `UIApplication+Notifications.swift`

Reply:
Ah! It was my first instinct too! I prefer that as well :)